scripts/legacy/simple_validation.py

```
@@ -1,16 +1,15 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)